app/shared/components/Wallet.js

```diff
@@ -9,6 +9,7 @@ import WalletStatus from './Wallet/Status';
 
 type Props = {
   actions: {},
+  actionObjects: {},
```

Comment:
`actionHistory` perhaps? `actionObjects` is a terrible name :)

Reply:
😃

Reply:
And so is `settings` and `balances` and `actions`, but you don't see the word `objects` appended to all of those.